packer/ubuntu16.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Rename this file to `ubuntu18.json` because you use `ubuntu18.json` in your documentation

Reply:
I've fixed it here https://github.com/loktionovam/otus-homeworks/commit/1dce3912bb39e01db16a283fcea90853b1a15525